mantidproject/mantid
# Improve Muon Analysis 2 single and simultaneous fitting

**Description of work.**
This PR is the first part of the planned improvements to the Muon Analysis fitting tab. The aim of the PR is make single, and simultaneous fits easier to undertake, especially for the users most common fitting requirements. 

Without the second part of fitting completed this PR represents a regression with regards to features, as sequential fitting has now been removed and will be re-added in the next PR. Therefore, the PR will be merged into a feature branch.

**To test:**
Verify the common workflow works as expected:
(1) Open the Muon Analysis 2 GUI
(2) Load in data, e.g MUSR 62260
(3) Go to fitting tab
(4) Add a fit function, select a workspace to do the fit over in the display workspace box
(5) Perform fit
(6) Tick the simultaneous box
(7) Cycle over the simultaneous fitting options and perform new fits, ensuring everything is okay.

More generally, make sure that the existing functionality within the GUI is not broken, and works as expected.



Fixes #27746. 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
